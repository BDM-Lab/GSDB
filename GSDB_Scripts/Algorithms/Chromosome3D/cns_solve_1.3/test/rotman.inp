remarks  file ROTMAN.INP --- test rotman facility 
remarks  Author: Axel T. Brunger


rotman

   euler=( 20. 30. 40. )
   ?

   copy 
   ?

   inverse
   ?

   product 
   ?

   swap
   ?

   distance

   allpairs


end

xray
   symmetry=( x, y, z )
   symmetry=( -x,-y,z+1/2)
end

rotman

   euler=( 105.23 102.33 54.764 )
   swap
   euler=( 17.901 72.826 227.67 )

   distance

   display  $DISTANCE


   display  $theta1  $theta2 $theta3 
   display  $rot_1_1, $rot_1_2, $rot_1_3
   display  $rot_2_1, $rot_2_2, $rot_2_3
   display  $rot_3_1, $rot_3_2, $rot_3_3
   display  $axis_x, $axis_y, $axis_z $axis_kappa
   display  $psi, $phi, $kappa
   display  $q0, $q1, $q2, $q3


end

stop
