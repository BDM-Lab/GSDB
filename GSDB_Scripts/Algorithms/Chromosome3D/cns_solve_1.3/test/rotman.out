          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:06 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks  file ROTMAN.INP --- test rotman facility 
 CNSsolve>remarks  Author: Axel T. Brunger 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>rotman 
 ROTMAN> 
 ROTMAN>   euler=( 20. 30. 40. ) 
 ROTMAN>   ? 
 ==============================================================================

 [symmetry operator No.  1] * [primary matrix]

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ==============================================================================

 [symmetry operator No.  1] * [secondary matrix]

 Rotation matrix =
    1.000000    0.000000    0.000000
    0.000000    1.000000    0.000000
    0.000000    0.000000    1.000000
 Corresp. Eulerian angles (theta1,theta2,theta3)     0.0000    0.0000    0.0000
 Corresp. spherical polar angles (psi,phi,kappa)     0.0000    0.0000    0.0000
 Corresp. rotation angle     0.0000 about axis       0.0000    1.0000    0.0000
 Corresp. quaternions     1.0000    0.0000    0.0000    0.0000

 ==============================================================================
 ROTMAN> 
 ROTMAN>   copy 
 ROTMAN>   ? 
 ==============================================================================

 [symmetry operator No.  1] * [primary matrix]

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ==============================================================================

 [symmetry operator No.  1] * [secondary matrix]

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ROTMAN> 
 ROTMAN>   inverse 
 ROTMAN>   ? 
 ==============================================================================

 [symmetry operator No.  1] * [primary matrix]

 Rotation matrix =
    0.529454   -0.830924    0.171010
    0.785102    0.403559   -0.469846
    0.321394    0.383022    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)   140.0000   30.0000  160.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232   66.4519
 Corresp. rotation angle    66.4519 about axis      -0.4652    0.0820   -0.8814
 Corresp. quaternions     0.8365   -0.2549    0.0449   -0.4830

 ==============================================================================
 ==============================================================================

 [symmetry operator No.  1] * [secondary matrix]

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ROTMAN> 
 ROTMAN>   product 
 ROTMAN>   ? 
 ==============================================================================

 [symmetry operator No.  1] * [primary matrix]

 Rotation matrix =
    1.000000    0.000000    0.000000
    0.000000    1.000000    0.000000
    0.000000    0.000000    1.000000
 Corresp. Eulerian angles (theta1,theta2,theta3)     0.0000    0.0000    0.0000
 Corresp. spherical polar angles (psi,phi,kappa)     0.0000    0.0000    0.0000
 Corresp. rotation angle     0.0000 about axis       0.0000    1.0000    0.0000
 Corresp. quaternions     1.0000    0.0000    0.0000    0.0000

 ==============================================================================
 ==============================================================================

 [symmetry operator No.  1] * [secondary matrix]

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ROTMAN> 
 ROTMAN>   swap 
 ROTMAN>   ? 
 ==============================================================================

 [symmetry operator No.  1] * [primary matrix]

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ==============================================================================

 [symmetry operator No.  1] * [secondary matrix]

 Rotation matrix =
    1.000000    0.000000    0.000000
    0.000000    1.000000    0.000000
    0.000000    0.000000    1.000000
 Corresp. Eulerian angles (theta1,theta2,theta3)     0.0000    0.0000    0.0000
 Corresp. spherical polar angles (psi,phi,kappa)     0.0000    0.0000    0.0000
 Corresp. rotation angle     0.0000 about axis       0.0000    1.0000    0.0000
 Corresp. quaternions     1.0000    0.0000    0.0000    0.0000

 ==============================================================================
 ROTMAN> 
 ROTMAN>   distance 
 ===following symmetry operators yield the minimum distance====================

 Matrix A = [symmetry operator No.  1] * [primary matrix]
 Matrix B = [symmetry operator No.  1] * [secondary matrix]

 Distance between matrices A and B=    1.5498
 Matrix A * B-1 

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ROTMAN> 
 ROTMAN>   allpairs 
 ==============================================================================

 Matrix A = [symmetry operator No.  1] * [primary matrix]
 Matrix B = [symmetry operator No.  1] * [secondary matrix]

 Distance between matrices A and B=    1.5498
 Matrix A * B-1 

 Rotation matrix =
    0.529454    0.785102    0.321394
   -0.830924    0.403559    0.383022
    0.171010   -0.469846    0.866025
 Corresp. Eulerian angles (theta1,theta2,theta3)    20.0000   30.0000   40.0000
 Corresp. spherical polar angles (psi,phi,kappa)    85.2952  117.8232  293.5481
 Corresp. rotation angle    66.4519 about axis       0.4652   -0.0820    0.8814
 Corresp. quaternions     0.8365    0.2549   -0.0449    0.4830

 ==============================================================================
 ROTMAN> 
 ROTMAN> 
 ROTMAN>end 
 CNSsolve> 
 CNSsolve>xray 
 XRAY>   symmetry=( x, y, z ) 
 XRAY>   symmetry=( -x,-y,z+1/2) 
 XRAY>end 
 CNSsolve> 
 CNSsolve>rotman 
 ROTMAN> 
 ROTMAN>   euler=( 105.23 102.33 54.764 ) 
 ROTMAN>   swap 
 ROTMAN>   euler=( 17.901 72.826 227.67 ) 
 ROTMAN> 
 ROTMAN>   distance 
 ===following symmetry operators yield the minimum distance====================

 Matrix A = [symmetry operator No.  1] * [primary matrix]
 Matrix B = [symmetry operator No.  2] * [secondary matrix]

 Distance between matrices A and B=    2.0298
 Matrix A * B-1 

 Rotation matrix =
    0.823060    0.275994   -0.496386
    0.477674    0.136415    0.867881
    0.307244   -0.951430   -0.019557
 Corresp. Eulerian angles (theta1,theta2,theta3)    17.8968   91.1206  330.2325
 Corresp. spherical polar angles (psi,phi,kappa)    66.2969    6.3257   91.7215
 Corresp. rotation angle    91.7215 about axis       0.9101    0.4020   -0.1009
 Corresp. quaternions     0.6964    0.6531    0.2885   -0.0724

 ==============================================================================
 ROTMAN> 
 ROTMAN>   display  $DISTANCE 
 2.02982
 ROTMAN> 
 ROTMAN> 
 ROTMAN>   display  $theta1  $theta2 $theta3 
 17.8968  91.1206 330.233
 ROTMAN>   display  $rot_1_1, $rot_1_2, $rot_1_3 
 0.82306, 0.275994, -0.496386
 ROTMAN>   display  $rot_2_1, $rot_2_2, $rot_2_3 
 0.477674, 0.136415, 0.867881
 ROTMAN>   display  $rot_3_1, $rot_3_2, $rot_3_3 
 0.307244, -0.95143, -1.955742E-02
 ROTMAN>   display  $axis_x, $axis_y, $axis_z $axis_kappa 
 0.910066, 0.401997, -0.100886 91.7215
 ROTMAN>   display  $psi, $phi, $kappa 
 66.2969, 6.32572, 91.7215
 ROTMAN>   display  $q0, $q1, $q2, $q3 
 0.696405, 0.653108, 0.288493, -7.240062E-02
 ROTMAN> 
 ROTMAN> 
 ROTMAN>end 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8201488 bytes
           Maximum dynamic memory overhead:           272 bytes
           Program started at: 16:08:06 on 05-Jul-2018
           Program stopped at: 16:08:06 on 05-Jul-2018
           CPU time used:       0.0050 seconds
          ============================================================
